<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Snoozer</class>
 <widget class="QDialog" name="Snoozer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="alarmLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>ALARM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="snoozeButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Snooze Alarm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="turnOffButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Turn Off Alarm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
